### DISCUSSION TITLE
using the %s (EPOCH modifier) in POST request

### DISCUSSION THREAD

**Discussion by MickSoundIntel** [AxisCommunications/discussions#206] [VAPIX] [answered]
Hi ,

I've set up an action rule to trigger a HTTPS recipient and want to use the EPOCH modifier %s, or %25 encoded.  I'm new to API in general. quesiont: will using %25s ( i believe i need to encode it using the Quert string suffix field in the action rule) be enough to generate a EPOCH in seconds when POST method it to another API?

**Comment by vivekatoffice** [comment, marked as answer]
Hi @MickSoundIntel ,
I tested this and found out that if you directly mention "%s" works for the payload

# Modifier (%s) details from overlay window
![Screenshot 2023-09-12 130239](https://github.com/AxisCommunications/discussions/assets/52032125/149333c7-8570-4deb-8a51-846b9d11c9c7)

# Action rules

![Screenshot 2023-09-12 131111](https://github.com/AxisCommunications/discussions/assets/52032125/d06e1bea-7535-45dc-8bfc-d5c6660c5d8a)

# Output

![Screenshot 2023-09-12 131054](https://github.com/AxisCommunications/discussions/assets/52032125/a420b92e-ab73-4a4d-8270-d2dc0574aeed)

**Reply by MickSoundIntel** [reply]
Hi Vivek, thanks for sharing your results! looks promising.
